go-controller/pkg/ovn/secondary_layer2_network_controller.go

```diff
@@ -62,6 +63,10 @@ func (h *secondaryLayer2NetworkControllerEventHandler) GetResourceFromInformerCa
 // RecordAddEvent records the add event on this given object.
 func (h *secondaryLayer2NetworkControllerEventHandler) RecordAddEvent(obj interface{}) {
 	switch h.objType {
+	case factory.PolicyType:
```

Comment:
all this could go into base_event_handler

Reply:
Also, a bit below, why for RecordUpdateEvent we are calling `h.baseHandler.recordAddEvent` instead of `h.baseHandler.recordUpdateEvent`? 

PTAL @tssurya for awareness

Reply:
yeah sounds like a copy paste snafu nice catch, should be add, update, delete respectively. I'll take care of a cleanup PR for that

Reply:
@tssurya I've addressed it with this PR, please take a look.